library/src/main/java/com/rengwuxian/materialedittext/MaterialEditText.java

```diff
@@ -331,6 +331,14 @@ public MaterialEditText(Context context, AttributeSet attrs, int style) {
     init(context, attrs);
   }
 
+  public String getString() {
+    return getText().toString().trim();
```

Comment:
you are free to do that on your side. what about if I need to get space the user has typed?